src/algorithms/optimization/peps_optimization.jl

```diff
@@ -257,13 +265,87 @@ function fixedpoint(
     return peps_final, env_final, cost, info
 end
 
-# Update PEPS unit cell in non-mutating way
-# Note: Both x and η are InfinitePEPS during optimization
+"""
+    peps_normalize(A::InfinitePEPS)
+
+Normalize the individual tensors in the unit cell of an `InfinitePEPS` such that they each
+have unit Euclidean norm.
+"""
+function peps_normalize(A::InfinitePEPS)
+    normalized_tensors = map(unitcell(A)) do t
+        return t / norm(t)
+    end
```

Comment:
```suggestion
    normalized_tensors = normalize.(unitcell(A))
```
This is defined in LinearAlgebra and TensorKit, but might need to be imported